### ISSUE TITLE
Telegram Voice Repeat to Discord Voice Channel?

### ISSUE THREAD

**Issue by computeronix** [42wim/matterbridge#1473]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
In Telegram you can activate a group voice feature where users can join in and chat

When this is is activated, it would be interesting to allow the bridge - telegram bot, to join in and listen and then repeat this to a Discord voice channel.

**Describe alternatives you've considered**
Haven't researched other options yet

**Additional context**
Basically the bridge allows for linking Discord and Telegram communities via chat/text, what about if we had a community wide voice session in Telegram and wanted our Discord users the ability to listen in and then ask questions via the text channel? (not sure if possible but just throwing this out there since the bot most likely wouldn't be able to repeat voices of everyone on Discord back to TG)

**Comment by xavierxross** [comment]
Telegram Bots can't voice chat, so need to use a dedicated telegram user account instead of bot. 

Btw sirji, it would be awesome if someone implement the ability bridge using telegram user-account besides bot in Matterbridge! The reason I'm asking is because telegram bots can't interact with other bots.I've some bots in Telegram; so yeah if bridged using user-bot then I would be able use those bots from Discord,whatsapp,irc!

**Comment by poVoq** [comment]
There is a Discord voice to Mumble voice bridge in case that helps.
